travis-ymls/jetty-reactive-httpclient.travis.yml

```diff
@@ -0,0 +1,28 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : jetty-reactive-httpclient 
+# Source Repo         : https://github.com/jetty-project/jetty-reactive-httpclient
+# Travis Job Link     : https://travis-ci.com/github/rajesh-ibm-power/jetty-reactive-httpclient
+# Created travis.yml  : Yes
+# Maintainer          : Rajesh kumar <Rajesh.kumar13@ibm.com>
+#
+# Script License      : Apache License, Version 2 or later
+#
+# ---
+language: java
+
+jdk:
+  - openjdk11
+  
+dist: xenial
+
+arch:
+  - amd64
+  - ppc64le
+
+before_install:
+  - export M2_HOME=$HOME/apache-maven-3.5.4
+  - if [ ! -d $M2_HOME/bin ]; then curl https://archive.apache.org/dist/maven/maven-3/3.5.4/binaries/apache-maven-3.5.4-bin.tar.gz | tar zxf - -C $HOME; fi
+  - export PATH=$M2_HOME/bin:$PATH
```

Comment:
We have maven pre- installed in ppc64le image. Please remove this installation. @rajesh-ibm-power